src/components/Aside/Aside.tsx

```diff
@@ -23,7 +27,13 @@ export default function Aside({
         "is-pinned": pinned === true,
       })}
     >
-      {children}
+      {!loading ? (
+        children
+      ) : (
+        <div className="loading">
```

Comment:
The loading indicator is only rendered in the content where data is missing so in the config case, only the left column gets a spinner and in this case, the same, the 'add a user' column would not get a spinner so this should be handled in the panel itself, not the aside.